index.html

```diff
@@ -1,54 +1,273 @@
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads; no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button multi">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/Snapshoot Portfolio.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Tonic</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+
+        <div class="card">
+
+          <img class="cardimg" src="./images/profesional.png" alt="">
+
+          <div class="grid-item-title-1">
+            <p class="title">Multi-Post Stories</p>
+            <div class="canopy">
+              <p>CANOPY</p>
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button multi">
+            <ul>
+              <li class = "html">HTML</li>
+              <li class = "css">CSS</li>
+              <li class = "js">JavaScript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See project</button></a>
+            </div>
+          </section>
+
+        </div>
+      </section>
+
+      <section class="aboutmyself">
+        <h2>About Myself</h2>
+
+        <article class="offer">
+          <p>
+            Hello I’m a software developer! I can
+            help you build a product , feature or
+            website. Look through some of my
+            work and experience! If you like what
+            you see and have a project you need
+            coded, don’t hesitate to contact me.
+          </p>
+        </article>
+        <section class="contact">
+          <p>LET’S CONNECT</p>
```

Comment:
- [ ] **[OPTIONAL]** Please consider using a `hx` here instead of a `p`. Remember to use `hx` for each element that look like header. This will make your code look much accessible📱 which will improve the user experience 😄